From a6f923b25b7857bef9694b7f47614c84b032748a Mon Sep 17 00:00:00 2001
From: ariel aviad <aaviad@habana.ai>
Date: Sun, 22 Dec 2024 16:56:29 +0200
Subject: [PATCH] [FSW-6867] palad: remove pldm bin and eld files

remove pldm bin and elf files because we moved them to
pald directory
we kept the files just to temperrary support both pldm
and palad

Change-Id: I185cd493d9488e965d42b74ed1b29904157de453
Signed-off-by: ariel aviad <aaviad@habana.ai>
---

diff --git a/Makefile b/Makefile
index b25c8e6..9317b40 100644
--- a/Makefile
+++ b/Makefile
@@ -6,11 +6,6 @@
 HL_DEV_SUBTYPE := $(shell echo $(HL_DEV) | cut -d'_' -s -f2,3)
 HL_DEV_ARCH := arch/$(shell echo $(HL_DEV) | cut -d'_' -f1)/$(HL_DEV_SUBTYPE)
 
-# TODO: remove this after switch to palad target
-ifeq ($(HL_TARGET), pldm)
-    HL_TARGET = palad
-endif
-
 # removed liblto_plugin.so plugin from flags.
 # after upgrading to Synopsys Arc toolchain 2023.03, ar error is observed when attempt to use
 # lto plugin. In addition, when building without lto plugin no change in binary is observed.
diff --git a/bin/gaudi2_arc/pldm/arcboot_rom.bin b/bin/gaudi2_arc/pldm/arcboot_rom.bin
deleted file mode 100644
index 707165c..0000000
--- a/bin/gaudi2_arc/pldm/arcboot_rom.bin
+++ /dev/null
Binary files differ
diff --git a/bin/gaudi2_arc/pldm/arcboot_rom.elf b/bin/gaudi2_arc/pldm/arcboot_rom.elf
deleted file mode 100644
index f51d261..0000000
--- a/bin/gaudi2_arc/pldm/arcboot_rom.elf
+++ /dev/null
Binary files differ
diff --git a/bin/gaudi3_arc/pldm/arcboot_rom.bin b/bin/gaudi3_arc/pldm/arcboot_rom.bin
deleted file mode 100644
index eba1646..0000000
--- a/bin/gaudi3_arc/pldm/arcboot_rom.bin
+++ /dev/null
Binary files differ
diff --git a/bin/gaudi3_arc/pldm/arcboot_rom.elf b/bin/gaudi3_arc/pldm/arcboot_rom.elf
deleted file mode 100644
index 2162103..0000000
--- a/bin/gaudi3_arc/pldm/arcboot_rom.elf
+++ /dev/null
Binary files differ
